lib/bundler/cli/config.rb

```diff
@@ -21,6 +21,8 @@ def run
         return
       end
 
+      return Bundler.ui.info(Bundler.settings[name]) if options[:parseable]
```

Comment:
I'm sorry, added in 2ba6079. [The failure](https://travis-ci.org/bundler/bundler/jobs/160111059#L608) is fixed in #4990.